lib/msal-react/src/Constants.ts

```diff
@@ -0,0 +1,8 @@
+/*
+ * Interaction type of the API
+ */
+export enum InteractionType {
```

Comment:
This should be added to browser (I think there is an existing enum that can be exported).